azure/components/db/setup.ftl

```diff
@@ -143,7 +141,57 @@
                         server.Reference
                     ]
                 /]
+                [#break]
+
+            [#case "mysql"]
+                [@createMySqlServer
+                    id=server.Id
+                    name=server.Name
+                    location=regionId
+                    createMode=createMode
+                    adminName=masterAccount
+                    adminSecret=masterSecret
+                    keyvaultId=keyVaultId
+                    skuName=sku.Processor
+                    version=engineVersion
+                    backupRetentionDays=solution.Backup.RetentionPeriod
+                    storageGB=solution.Size
+                    storageAutogrow=solution.azure\:AutoGrow
```

Comment:
```suggestion
                    storageAutogrow=solution["azure:AutoGrow"]
```

That's the standard we mostly work for with other key lookups. The escape is a bit of a surprsie

Reply:
Sorry, just plagiarized it from the Azure postgres. I'll update both